pkg/cmd/version/version.go

```diff
@@ -0,0 +1,33 @@
+package version
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/spf13/viper"
+)
+
+//TODO move Version struct into go-mod-core-contracts
+type Version struct {
+	Version string `json:"version" yaml:"version,omitempty"`
+}
+
+// GetVersion returns the version of core-data microservice
+func GetVersion(port string) (version Version, err error) {
+	host := viper.GetString("dataservice.host")
+	url := "http://" + host + ":" + port + "/api/version"
```

Comment:
Just to note: `url := "http://" + host + ":" + port + "/api/version"`
Here, I am not useing "net.url" package. I just construct the URL string 
on the next line inside `http.Get(url)` `net.url.Parse` function is called - to validate that the URL is valid one. No need to do it twice.

Reply:
perfect. a clean up task for the future.